core/os/android/adb/perfetto.go

```diff
@@ -29,9 +32,24 @@ import (
 	"github.com/google/gapid/core/log"
 )
 
+// DeleteFile deletes a file on the device.
+func (b *binding) DeleteFile(ctx context.Context, traceFile string) error {
+	_, err := b.Shell("rm", "-f", traceFile).Call(ctx)
+	if err != nil {
+		return log.Errf(ctx, err, "Failed to delete perfetto trace file %v. Try root.", traceFile)
```

Comment:
`log.Errf(ctx, err, "Failed to delete file %v. Try root.", filePath)`, sorry, didn't notice this should be more generic. I feel like we should move this method to core/os/android/adb/command.go.